app/views/admin/users/show/notes/_index.html.erb

```diff
@@ -1,16 +1,16 @@
 <div class="flex flex-col h-100">
-  <h2 class="crayons-subtitle-1">Notes (<%= @notes.count %>)</h2>
+  <h2 class="crayons-subtitle-1">Previous notes (<%= @notes.count %>)</h2>
   <% if @notes.load.empty? %>
     <div class="align-center flex flex-col justify-center my-auto py-7">
-      <h3 class="fs-xl fw-normal mb-2">🗒 No notes found.</h3>
-      <p class="color-secondary">Any notes added for @<%= @user.name %> will be found here. </p>
+      <h3 class="fs-xl fw-normal mb-2">No previous notes found for <%= @user.name %>.</h3>
+      <p class="color-secondary">Any notes added for <%= @user.name %> will be found here. </p>
     </div>
   <% else %>
     <div class="pt-3">
       <% @notes.each do |note| %>
         <% author = User.find(note.author_id).username %>
```

Comment:
Reading note, outside of scope, it would make sense for this to be `note.author.username`.  This creates an N+1 query situation.

Non-blocking just an observation.